cmd/topicctl/subcmd/get.go

```diff
@@ -38,109 +32,249 @@ type getCmdConfig struct {
 var getConfig getCmdConfig
 
 func init() {
-	getCmd.Flags().BoolVar(
+	getCmd.PersistentFlags().BoolVar(
```

Comment:
`PersistentFlags` allows us to pass these flags through from the parent command (`get`) to all the child commands